docs/docs/dev/documentation/dps-and-data-model.md

```diff
@@ -125,7 +125,7 @@ Each Kuma data plane proxy is associated with tags - or attributes - that can be
 A tag attributes a qualifier to the data plane proxy, and the tags that are reserved to Kuma are prefixed with `kuma.io` like:
 
 * `kuma.io/service`: Identifies the service name. On Kubernetes this tag is automatically created, while on Universal it must be specified manually.
-* `kuma.io/zone`: Identifies the zone name in a [multi-zone deployment](../../deployments/how-multi-zone-works). This tag is automatically created and cannot be overwritten.
+* `kuma.io/zone`: Identifies the zone name in a [multi-zone deployment](../deployments/multi-zone.md). This tag is automatically created and cannot be overwritten.
```

Comment:
should we always use `.md` then? I was recently getting rid of this...

Reply:
I don't know TBH it feels like when you use md the path is relative to the directory tree. When you don't use it it's relative to the whole docs generated tree (think table of content). So it seems .md is easier to use.